src/modules/emote_menu/styles/menu.module.css

```diff
@@ -0,0 +1,37 @@
+.divider {
+  margin: 0px;
+}
+
+.header {
+  display: flex;
+  padding: 8px;
+  column-gap: 4px;
+}
+
+.content {
+  display: flex;
+  height: 308px;
+}
+
+.emojis {
+  width: 100%;
+  padding: 0 10px;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  scrollbar-width: thin;
+  scrollbar-color: #191c23 transparent;
+}
+
+.emojis::-webkit-scrollbar {
+  border-radius: 6px;
```

Comment:
rather than copy/pasting scrollbar styles, can we standardize these? you can use `composes`

Reply:
can you also explain why this wrapping component has a scrollbar, and the emojis container also has a scrollbar? nesting scrollbars sounds like a bug